src/commands/deploy.ts

```diff
@@ -189,7 +189,7 @@ export const command = (async (ctx, firstRun = true) => {
         for (const issue of issues) {
           const message = issue.message.replace(/"/g, "");
```

Comment:
```suggestion
          const message = issue.message.replace(/[".]/g, "");
```
Small nit, noticed some messages have an ending "`.`" (as well as the double quotes), we can remove it if we want the messages formatted consistently
<img src="https://github.com/gadget-inc/ggt/assets/92458251/fe43f6b1-0499-4d4a-bec2-425a7b803dcb" width="300"/>
<img src="https://github.com/gadget-inc/ggt/assets/92458251/de9f6312-10a0-4192-ba42-1cd94892ad18" width="300" />